supervisor/parser.go

```diff
@@ -74,16 +78,19 @@ func ParseOption(c *caddy.Controller, options *Options) {
 				options.TerminationGracePeriod = period
 			} else {
 				log.Printf("Invalid 'termination_grace_period' value %v\n", args[0])
+				return false
 			}
 		} else {
 			log.Printf("Option 'termination_grace_period' expects 1 argument\n")
+			return false
 		}
 	case "env":
 		args := c.RemainingArgs()
-		if len(args) == 1 {
-			options.Env = append(options.Env, args[0])
+		if len(args) > 0 {
```

Comment:
Instead of changing it to join multiple args. It might be better to add quotes on your tests:
```
NUMBER="{{ add 1 .Replica }}"
```

Reply:
Hey there, that was actually my first try when I was trying to use this plugin with environment variables.
The parser does not use the `"` to interpret it as a single value.
<img width="247" alt="image" src="https://user-images.githubusercontent.com/51510/69810860-1a09e980-11ed-11ea-9efa-b331bfeb554f.png">

Reply:
Found the issue, quotes only work after whitespace, at the beginning of a new token...
This would be the correct way:
```
env "NUMBER={{ add 1 .Replica }}"
```